dependencies.svg

```diff
@@ -4,15 +4,15 @@
 <!-- Generated by graphviz version 2.49.2 (20211016.1639)
  -->
 <!-- Pages: 1 -->
-<svg width="230pt" height="260pt"
- viewBox="0.00 0.00 230.24 260.00" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+<svg width="287pt" height="260pt"
```

Comment:
Added a note along these lines. I'd usually prefer a pre-commit hook, but I would like to avoid breaking pre-commit on systems without `dot` installed.

Reply:
I think that makes a lot of sense